src/app/clusters/groups-server/groups-server.cpp

```diff
@@ -283,7 +283,7 @@ bool emberAfGroupsClusterRemoveGroupCallback(app::CommandHandler * commandObj, c
 
 #ifdef EMBER_AF_PLUGIN_SCENES
     // If a group is, removed the scenes associated with that group SHOULD be removed.
```

Comment:
```suggestion
    // If a group is removed the scenes associated with that group SHOULD be removed.
```
I know, preexisting, followup fine for this.  Just noticed it.